dap-go.el

```diff
@@ -0,0 +1,97 @@
+;;; dap-go.el --- Debug Adapter Protocol mode for Go      -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2018  Aya Igarashi
+
+;; Author: Aya Igarashi <ladiclexxx@gmail.com>
+;; Keywords: languages
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+;; URL: https://github.com/yyoncho/dap-mode
+;; Package-Requires: ((emacs "25.1") (dash "2.14.1") (lsp-mode "4.0"))
+;; Version: 0.2
+
+;;; Commentary:
+;; Adapter for https://github.com/Microsoft/vscode-go
+
+;;; Code:
+
+(require 'dap-mode)
+
+(defcustom dap-go-debug-program `("node" ,(expand-file-name "~/.extensions/go/out/src/debugAdapter/goDebug.js"))
```

Comment:
Thanks for your awesome job. I'm a very new user of Visual Studio Code. I haven't find this file after vscode-go plugin installed. But I do found the same usage file at `~/.vscode/extensions/ms-vscode.go-0.9.2/out/src/debugAdapter/goDebug.js`.

Hope the below information may help:

- Visual Studio Code 1.32.1
- vscode.go 0.9.2

After added below line in my `init.el`, it works:

```lisp
(setq dap-go-debug-program `("node" ,(expand-file-name "~/.vscode/extensions/ms-vscode.go-0.9.2/out/src/debugAdapter/goDebug.js")))
```